packages/portfolio-contract/test/cctp-resolver.test.ts

```diff
@@ -4,42 +4,32 @@ import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 import { E } from '@endo/far';
 import { mustMatch } from '@endo/patterns';
 import type { TransactionSettlementOfferArgs } from '../src/resolver/types.ts';
-import { ResolverOfferArgsShapes } from '../src/resolver/types.ts';
+import { TransactionSettlementOfferArgsShape } from '../src/resolver/types.ts';
 import { deploy } from './contract-setup.ts';
 
-test('CCTP settlement invitation - no pending transaction found', async t => {
```

Comment:
To demonstrate that this is a compatible change, I'd rather leave the old test in place and _add_ a test that uses invoke.

Reply:
All of this test was recapitulated by the `CCTP confirmation invitation exits seat properly` test, which does extra checks, and was kept in place.